include/glow/IR/GraphScheduler.h

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_IR_GRAPH_SCHEDULER_H
```

Comment:
include/glow/ ... is a public include directory (headers used by multiple libraries).
Do you plan to have any consumer of `Scheduler` outside of the `lib/IR/GraphScheduler.cpp`?
If not, consider moving header to the lib/IR

Reply:
@opti-mix mentioned that Scheduler is going to be used in tests. So this is fine.

Reply:
Nice @qcolombet! This sounds better.